src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/ConfirmRolesAndResponsibilitiesSteps.cs

```diff
@@ -15,15 +15,15 @@ public sealed class ConfirmRolesAndResponsibilitiesSteps
         private readonly Fixture _fixture = new Fixture();
         private readonly TestContext _context;
         private readonly Apprentice _apprentice;
-        private readonly Apprenticeship _apprenticeship;
+        private readonly CommitmentStatement _apprenticeship;
         private bool? RolesAndResponsibilitiesCorrect { get; set; }        
 
         public ConfirmRolesAndResponsibilitiesSteps(TestContext context)
         {
             _context = context;
 
             _apprentice = _fixture.Create<Apprentice>();
-            _apprenticeship = _fixture.Create<Apprenticeship>();
+            _apprenticeship = _fixture.Create<CommitmentStatement>();
```

Comment:
minor but should we rename _apprenticeship to _commitmentStatement

Reply:
Yep, good idea.

Reply:
I have renamed it